package.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "gpii-devpmt",
+    "productName": "devpmt",
+    "version": "1.0.0",
+    "description": "Developers Preference Testing Tool",
+    "main": "devpmt.js",
+    "scripts": {
+        "start": "node devpmt.js",
+        "watch-sass": "gulp",
+        "build-sass": "gulp sass",
+        "pretest": "node node_modules/rimraf/bin.js reports/* coverage/*",
+        "test:node": "node node_modules/nyc/bin/nyc.js node tests/all-tests.js",
+        "test": "npm run test:node",
+        "posttest": "node node_modules/nyc/bin/nyc.js report -r text-summary -r html --report-dir reports --temp-directory coverage"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/sgithens/gpii-devpmt"
+    },
+    "author": "Steven Githens <sgithens@raisingthefloor.org>",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.gpii.net/"
+    },
+    "homepage": "https://github.com/sgithens/gpii-devpmt",
+    "dependencies": {
+        "connect-redis": "3.3.3",
+        "express-session": "1.15.6",
+        "file-saver": "1.3.8",
+        "foundation-sites": "6.4.3",
+        "gpii-express-user": "sgithens/gpii-express-user#gpii-express-1.0.15",
+        "gpii-handlebars": "sgithens/gpii-handlebars#GPII-3698",
+        "gpii-universal": "0.3.0-dev.20190131T192812Z.7eb9cba7",
+        "gulp-autoprefixer": "3.1.1",
+        "handlebars-helpers": "0.8.0",
+        "json5": "1.0.1",
+        "jsonpointer.js": "0.4.0",
+        "lunr": "2.0.3"
+    },
+    "devDependencies": {
+        "browserify": "16.2.3",
+        "gpii-express": "1.0.15",
+        "grunt": "1.0.3",
+        "grunt-markdownlint": "2.1.0",
+        "http-browserify": "1.7.0",
+        "https-browserify": "1.0.0",
+        "nyc": "13.1.0",
+        "pouchdb": "7.0.0",
```

Comment:
Thankfully nothing needs to depend on pouchdb any more, please remove this dependency

Reply:
removed.